packages/fxa-content-server/app/tests/spec/views/settings/change_password.jsx

```diff
@@ -125,6 +125,12 @@ describe('views/settings/change_password', function () {
       });
     });
 
+    describe('handleSubmit', () => {
```

Comment:
heh 'handleSubmit' might be a little misleading since that's not called in the test.  (But feel free to keep it; not a deal breaker.)

Reply:
Hrm yeah, the validation happens in that method, but would make more sense with more coverage.